lib/config-generator.js

```diff
@@ -318,8 +318,12 @@ class ConfigGenerator {
         }
 
         if (this.webpackConfig.useEslintLoader) {
+            const test = this.webpackConfig.useVueLoader
```

Comment:
I'm wondering if that test is needed... would it be a bad thing if we always included `.vue` files? If the vue loader isn't enabled (and not added manually) the build would fail anyway right?

If we always include `.vue` file:
* it makes our code a bit simpler
* if for some reason someone chooses not to call `enableVueLoader` but set-up its own loader manually they can still use our default ESLint config

Reply:
> If the vue loader isn't enabled (and not added manually) the build would fail anyway right?

Yes, I think ESLint will throw an error like this if the user didn't have configurer eslint-plugin-vue: 
> Unexpected token &lt;template&gt; ....


A safer way is to let the user configure ESLint loader:

```js
Encore
  .enableVueLoader()
  .enableEslintLoader()
  .configureEslintLoader(loader => {
    loader.test = /.(jsx?|vue)$/
  })
```

It's a bit more verbose, but it will prevent some issues.

Reply:
Or maybe a more generic method, if we don't want to add one method for each loader: 

```js
Encore
  .configureLoader('eslint', loader => {
    loader.test = /.(jsx?|vue)$/
  });

Reply:
> Yes, I think ESLint will throw an error like this if the user didn't have configurer eslint-plugin-vue

Hm, yes but that's only if you try to import a `.vue` file right?

What I meant is that if you don't enable the `vue-loader` the build will fail because Webpack won't know how to process those files anyway.

And if you don't use `.vue` files, well, that's not an issue because you'll never get that error.

This raises a point though... should we force people to use the Eslint vue plugin by default if they have `.vue` files?

Reply:
> Hm, yes but that's only if you try to import a .vue file right?

Yep

> What I meant is that if you don't enable the vue-loader the build will fail because Webpack won't know how to process those files anyway.
>
> And if you don't use .vue files, well, that's not an issue because you'll never get that error.

Hum you're right, I didn't think this way.

> This raises a point though... should we force people to use the Eslint vue plugin by default if they have .vue files?

I don't know. I'm probably sure that some people will complain about that. :thinking: 

But if we do, we should not hardcode eslint-plugin-vue configuration inside Encore.